src/Compilers/CSharp/Test/Symbol/Symbols/GenericConstraintTests.cs

```diff
@@ -6328,17 +6340,14 @@ public struct S
  
 public class E { }
 ";
-
+            // Note: error types are assumed to be value types when annotated with `?`
```

Comment:
>// Note: error types are assumed to be value types when annotated with `?` [](start = 12, length = 74)

Are there error types in this test?

Reply:
Removed. Thanks

---
In reply to: [209720840](https://github.com/dotnet/roslyn/pull/29225#discussion_r209720840) [](ancestors = 209720840)